libcst/metadata/scope_provider.py

```diff
@@ -54,7 +57,7 @@ def __new__(cls) -> "Tree":
 
     #: The name node of the access. A name is an access when the expression context is
     #: :attr:`ExpressionContext.LOAD`.
-    node: cst.Name
+    node: Union[cst.Name, cst.Attribute]
```

Comment:
Can you also update the inline comments to explain in what situation the node will be an `Attribute`?

Reply:
done!